### ISSUE TITLE
Images not loading

### ISSUE THREAD

**Issue by willtack** [neurosynth/nsviewer#27]
Hello there!

Looks like there hasn't been much activity here in awhile, but I was looking to use this viewer as part of an [html report generator](https://github.com/willtack/presurgical-report) I'm working on. Everything seems to work fine, except loading the images. When I cloned the repo and opened index.html in the example, the images wouldn't load. The spherical ROI does load, so it seems like app.js isn't finding the images. I tried using absolute paths to no avail. 

Let me know if anyone has any ideas or clarifying questions, etc

**Comment by tyarkoni** [comment]
@willtack yeah, sorry, this project is more or less dead... I'll still consider PRs, but am not doing any maintenance otherwise. If you're building a report generator, I'd suggest looking into other better-developed viewers, e.g. [Papaya..js](https://github.com/rii-mango/Papaya).
